excellent/functions.go

```diff
@@ -1610,6 +1613,44 @@ func Now(env utils.Environment, args ...interface{}) interface{} {
 	return time.Now().In(env.Timezone())
 }
 
+//----------------------------------------------------------------------------------------
+// URN Functions
+//----------------------------------------------------------------------------------------
+
+// FormatURN turns `urn` into a human friendly string
+//
+//   @(format_urn("tel:+250781234567")) -> 0781 234 567
+//   @(format_urn("twitter:134252511151#billy_bob")) -> billy_bob
+//   @(format_urn("NOT URN")) -> ERROR
+//
+// @function format_urn(urn)
+func FormatURN(env utils.Environment, args ...interface{}) interface{} {
+	if len(args) != 1 {
+		return fmt.Errorf("FORMAT_URN takes one argument, got %d", len(args))
+	}
+
+	// if we've been passed a slice like a URNList, use first item
+	urnArg := args[0]
+	if utils.IsSlice(urnArg) {
+		sliceLen, _ := utils.SliceLength(urnArg)
+		if sliceLen >= 1 {
+			urnArg, _ = utils.LookupIndex(urnArg, 0)
+		}
```

Comment:
Maybe 0 case should be explicitly handled here? Either as an error or an empty string? (I lean towards empty string though could be easily swayed to it being an error)

Reply:
I prefer empty string so that `@contact.urns` and `@(format_urn(contact.urns))` both evaluate to empty (when stringified)